Test/git-issues/git-issue-283.dfy

```diff
@@ -0,0 +1,16 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype Result<T> =
+    | Success(value: T)
```

Comment:
Use an indent of 2 spaces throughout this file

Reply:
Done